docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -132,6 +132,15 @@ Dates listed indicate the start date of the maintenance cycle.
      - End Of Life
      - Controller Python
      - Target Python / PowerShell
+   * - `2.16`
+     - | GA: 06 Nov 2023
+       | Critical: 20 May 2024
+       | Security: Nov 2024
+     - May 2030
+     - | Python 3.10 - 3.12
+     - | Python 2.7
+       | Python 3.6 - 3.12
+       | Powershell TBD
```

Comment:
These changes should be in a separate PR, since they're related to the 2.16 release and not the Python version removal. It's likely we'll want to backport this change.

Reply:
older branches are redirected to devel's release and maintenance page (fixing https://github.com/ansible/ansible/issues/71468), so I don't think this will need to be backported, @samccann does that sound right to you?

Reply:
Starting in 2.13, I began backporting all relevant docs PRS so the docs no longer point people to devel for the release/maint page etc. People really didn't like being pointed to devel. So I keep the current release (2.15 now) in sync with devel. Well, I have a ton of backports to do, but within a week or so I should have that all caught up.  So yes, separating out a 2.16 docs change would be preferable here.

Also, has there been an announcement to the community about these 2.16 python changes? Something in the bullhorn and/or ..erm.. maintainer news?

Reply:
@samccann Not yet, announcement will be made once it's approved.